docs/reference_guides/model_libraries/gas_solid_contactors/unit_models/fixed_bed_1D.rst

```diff
@@ -0,0 +1,348 @@
+   ":math:`T_{s,t,x}`", "Solid phase temperature", "``solid_phase.properties.temperature``"
+   ":math:`u_{g,t,x}`", "Superficial velocity of the gas", "``velocity_superficial_gas``"
+   ":math:`u_{s,t}`", "Superficial velocity of the solids", "``velocity_superficial_solid``"
+   "*Greek letters*", " ", " "
+   ":math:`\varepsilon`", "Reactor bed voidage", "``bed_voidage``"
+   ":math:`\mu_{g,t,x}`", "Dynamic viscosity of gas mixture", "``gas_phase.properties.visc_d``"
+   ":math:`\xi_{g,t,x,r}`", "Gas phase reaction extent", "``gas_phase.rate_reaction_extent``"
+   ":math:`\rho_{mass,g,t,inlet}`", "Density of gas mixture", "``gas_phase.properties.dens_mass``"
+   ":math:`\rho_{mass,s,t,inlet}`", "Density of solid particles", "``solid_phase.properties.dens_mass_particle``"
+   ":math:`\rho_{mol,g,t,x}`", "Molar density of the gas", "``gas_phase.properties.dens_mole``"
+
+List of Parameters
+------------------
+
+.. csv-table::
+   :header: "Parameter", "Description", "Reference to"
+
+   ":math:`d_{p}`", "Solid particle diameter", "``solid_phase.properties._params.particle_dia``"
+   ":math:`\nu_{s,j,r}`", "Stoichiometric coefficients", "``solid_phase.reactions.rate_reaction_stoichiometry``"
+
+
+Initialization
+--------------
+
+The initialization method for this model will save the current state of the model before
+commencing initialization and reloads it afterwards. The state of the model will be the same after
+initialization, only the initial guesses for unfixed variables will be changed.
+
+The model allows for the passing of a dictionary of values of the state variables of the gas and
+solid phases that can be used as initial guesses for the state variables throughout the time and
+spatial domains of the model. This is optional but recommended. A typical guess could be values
+of the gas and solid inlet port variables at time :math:`t=0`.
+
+The model initialization proceeds through a sequential hierarchical method where the model
+equations are deactivated at the start of the initialization routine, and the complexity of the model
+is built up through activation and solution of various sub-model blocks and equations at each
+initialization step. At each step the model variables are updated to better guesses obtained from
+the model solution at that step.
+
+The initialization routine proceeds as follows:
+
+*  Step 1:  Initialize the thermo-physical and transport properties model blocks.
+*  Step 2:  Initialize the hydrodynamic properties.
+*  Step 3a: Initialize mass balances without reactions and pressure drop.
+*  Step 3b: Initialize mass balances with reactions and without pressure drop.
+*  Step 3c: Initialize mass balances with reactions and pressure drop.
+*  Step 4:  Initialize energy balances.
+
+
+Block Triangularization Initialization
+--------------------------------------
+
+The Block Triangularization initialization routine initializes the 1DFixedBed model in the following steps:
```

Comment:
Is there any place in the IDAES documentation about this “Triangularization Initialization”. If so, it would be great to have a link here

Reply:
No there is no link. I put this together for this model based on work Robert Parker did for his PhD thesis.